src/gui/tray/activitydata.cpp

```diff
@@ -174,7 +174,7 @@ OCC::Activity Activity::fromActivityJson(const QJsonObject &json, const AccountP
     }
 
     auto actions = json.value("actions").toArray();
-    foreach (auto action, actions) {
+    for (auto action : actions) {
```

Comment:
We might as well constify this IMO

```suggestion
    for (const auto action : actions) {
```

Reply:
I would also constify actions